src/structures/Member.ts

```diff
@@ -0,0 +1,18 @@
+import type { User } from './User';
+
+export class Member {
+	public constructor(
+		public roles: null,
+		public premiumSince: string | null,
+		public pending: boolean,
+		public nick: string | null,
+		public mute: boolean,
+		public joinedAt: Date,
+		public hoistedRole: string,
+		public flags: number,
+		public deaf: boolean,
+		public communicationDisabledUntil: string | null,
+		public avatar: string | null,
+		public user: User | null
+	) {}
```

Comment:
maybe make this MemberOptions as an interface instead and do assigning

Reply:
yeah this is very outdated since our new format
example:
```ts
import type { APIUser } from 'discord-api-types/v10';

export class User {
	public id = this.raw.id;

	public username = this.raw.username;
	public discriminator = this.raw.discriminator;

	public avatar = this.raw.avatar;
	public banner = this.raw.banner;

	public bot = this.raw.bot;
	public verified = this.raw.verified;
	public system = this.raw.system;

	public constructor(private raw: APIUser) {}
}
```